views/layout/movieRender.ejs

```diff
@@ -1,15 +1,13 @@
-        <% if (Array.isArray(movieRender)) { %>
-          <% movieRender.forEach(movie => { %>
-        <div class="movieTemplate__card">
-          <div class="movieTemplate__cover" style="background-image:url(<%= movie.image.url%>)"></div>
-          <div class="movieTemplate__img" style="background-image:url(<%= movie.image.url%>)"></div>
-          <h2 class="movieTemplate__title"><%= movie.title  %></h2>
-          <a class="movieTemplate__button" href="/movies/<%- movie.id %>">
-            Se mer
+        
+        <% if (typeof movieRender === 'string' || movieRender instanceof String ){ %>
+          <div class="notFound">
+          <h1 class="notFound__title">Filmen kunde inte hittas! </h1>
+          <h2 class="notFound__error"><%- movieRender %></h2>
```

Comment:
The message you're printing here is coming from inside of the fetching code, which means that you are mixing the responsibility of UI and fetching. If you're going to hard-code the 404 message ("404 (Not found)") somewhere, you might as well hard-code it here in the template, which is where UI belongs.